### PR TITLE
Adding chore template

### PR DESCRIPTION
**Pull request by ben851**
Creating a chore template for us to manage notify core chores.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/core-chore-template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Core Chore Template
+about: Use this template for Notify Core chores
+title: "[CHORE] "
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/core-chore-template.md:4]:
```suggestion
title: "[CHORE] TASK NAME"
```

File: .github/ISSUE_TEMPLATE/core-chore-template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+labels: Chore, Routine, Tech Debt
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/core-chore-template.md:5]: I think these would refer to existing labels we have. We could create Routine or Chore, I would just keep one of these and not both.

Comment (jimleroyer) [reply .github/ISSUE_TEMPLATE/core-chore-template.md:5]: Ok I see we have both existing, but Chore refers to no issue and Routine does. I would like to delete Chore then, what do you think?

Comment (jimleroyer) [reply .github/ISSUE_TEMPLATE/core-chore-template.md:5]:
```suggestion
labels: Routine, Tech Debt
```

File: .github/ISSUE_TEMPLATE/core-chore-template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+assignees: ''
+
+---
+
+## What
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/core-chore-template.md:10]:
```suggestion
# What
```

File: .github/ISSUE_TEMPLATE/core-chore-template.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Provide one or two sentences of what the chore is.
+
+## Why
+
+Describe why we are doing this chore.
+
+## Acceptance Criteria
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/core-chore-template.md:18]:
```suggestion
# Acceptance Criteria
```

File: .github/ISSUE_TEMPLATE/core-chore-template.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+-[ ] Provide a list of things that should be required in order to consider this complete
+
+## Resources
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/core-chore-template.md:22]:
```suggestion
# Resources
```

File: .github/ISSUE_TEMPLATE/core-chore-template.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/core-chore-template.md:23]:
```suggestion

# Triggers / Schedule

Describe the schedule or trigger for this chore. When should it be done?

```

File: .github/ISSUE_TEMPLATE/core-chore-template.md (hunk 2 above)
Comment (ben851) [reply .github/ISSUE_TEMPLATE/core-chore-template.md:5]: I created chore not seeing routine - you're right though. I'll delete Chore.

**Review by sastels — APPROVED** [review]
LGTM

**Pull request merged**